### ISSUE TITLE
[mdoc] Missing the params in signatures

### ISSUE THREAD

**Issue by dend** [mono/api-doc-tools#78]
As an example, compare [WhenAll](https://docs.microsoft.com/en-us/dotnet/api/System.Threading.Tasks.Task.WhenAll?view=netframework-4.7#System_Threading_Tasks_Task_WhenAll_System_Threading_Tasks_Task___) on docs and [WhenAll](https://msdn.microsoft.com/en-us/library/hh160374(v=vs.110).aspx?cs-save-lang=1&cs-lang=csharp#code-snippet-1) on MSDN.

Notice that on docs we have:

```
public static System.Threading.Tasks.Task WhenAll (System.Threading.Tasks.Task[] tasks);
```

On MSDN:

```
public static Task WhenAll(
	params Task[] tasks
)
```

Looks like the `params` keyword is getting lost.

**Comment by meganbradley** [comment]
@dend @joelmartinez Is there any ETA for this?
CC @mairaw

**Comment by joelmartinez** [comment]
@meganbradley I'm actually working on some improvements to our signature generation right now (fixed #82 last week, and working on #59 now). This issue will be in the same area of code, so let me see if I can include it in the next release :)

**Comment by dend** [comment]
@joelmartinez was there any progress on this specific issue, given that both #82 and #59 are resolved?

**Comment by joelmartinez** [comment]
Hi @dend, I unfortunately haven't had a chance to work on this one. I'm wrapping up the final fix to #59 right now (putting together a build that you'll have a bit later). I will probably work on this particular issue next week
